SPECS/rubygem-minitest/rubygem-minitest.spec

```diff
@@ -0,0 +1,53 @@
+%global debug_package %{nil}
+%global gem_name minitest
+Summary:        Minitest provides a complete suite of testing facilities
+Name:           rubygem-minitest
+Version:        5.13.0
+Release:        2%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://github.com/seattlerb/minitest
+# The upstream source doesn't contain gemspec file. This source has been taken from ruby-2.7.4.tar.xz
+Source0:        %{gem_name}-%{version}.tar.gz
```

Comment:
Could you also explain how to retrieve it for version updates? And since we're already on version 3.1.2 of `ruby`, why use `2.7.4`?

Reply:
I was taking ruby-minitest source from ruby-27.4 due to missing .gemspec in the upstream source. Fixed it to take source from upstream, and initial .gemspec from ruby